news/10482.bugfix.rst

```diff
@@ -0,0 +1 @@
+New resolver: Fix prefering packages which are not as deep in the dependency tree
```

Comment:
```suggestion
New resolver: Fix prefering packages which are not as deep in the dependency tree.
```

The sentence doesn’t feel very meaningful to me as a changelog though. What does “not as deep” mean? What difference should end users expect? (This is most important for a changelog.)

Reply:
I agree, but I am stumped on how to explain this in a meaningful way to end users without drawing out a graph.

Reply:
Maybe just simply mention this fixes ordering of packages during resolution (and when)?